src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -616,6 +638,25 @@ app.use(function processItems(req, res, next) {
 });
 ```
 
+#### Retrieving a Transaction
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry.getCurrentHub().getScope().getTransaction()`. This function will return a `Transaction` in case there is a running Transaction otherwise it returns `undefined`. If you are using our Express integration by default we attach the Transaction to the Scope. So you could do something like this:
+
+```javascript
+app.get("/success", function successHandler(req, res) {
+  const transaction = Sentry.getCurrentHub().getScope().getTransaction();
+  if (transaction) {
+    let span = transaction.startChild({
+      op: "encode",
+      description: "parseAvatarImages"
+    });
+    // Do something
+    span.finish();
```

Comment:
In Python we have context managers. The way they work guarantees that even if there is an exception in the middle of a span, the span will be finished.

Without `withTransaction` and `withSpan` wrappers, it might be a good idea to document using `try...finally` and calling `finish()` in the `finally` block.